.travis.yml

```diff
@@ -16,14 +16,8 @@ before_script:
   - chmod +x ./cc-test-reporter
   - ./cc-test-reporter before-build
 script:
-  - lerna run tslint
-  - lerna run unit-tests --maxWorkers=4
-  - lerna run build:production
-  - lerna run build:dist
-  - lerna run build:app
-  - lerna run build:interface
-  - lerna run build:chrome
-  - lerna run eslint
+  - lerna run test --stream
```

Comment:
Why was it decided to reset this? You'll need to travis-wait to fix the timeout issue:
`- travis_wait lerna run test --stream`

Reply:
finished in 10 minutes (60% faster). 

Moved workers to each package.json and streaming the entire thing.

Reply:
Ok I'll rebase and see if this works on my PR